cuda/matrix/sparsity_csr_kernels.cu

```diff
@@ -41,6 +41,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <ginkgo/core/matrix/dense.hpp>
 
 
+#include "core/components/fill_array.hpp"
```

Comment:
This also looks unused. Like most headers here actually since nothing is implemented.